source/class/qx/lang/Type.js

```diff
@@ -52,15 +52,23 @@ qx.Bootstrap.define("qx.lang.Type", {
     isArray: qx.Bootstrap.isArray,
 
     /**
-     * Whether the value is an object. Note that built-in types like Window are
-     * not reported to be objects.
+     * Whether the value is an object i.e. Object.prototype is its prototype or Object.prototype is in its prototype chain.
```

Comment:
it would be helpful to explain the net effect - eg:  "Objects can be either a POJO (ie `{}`) or an object which is created from a ES6-style class or prototypical-inheritance-based class; if you need to determine whether something is a POJO and _not_ created from a class, use `isPojo` instead"